Don't build EsounD support since it was deprecated years ago
by Gnome. This is a Debian build system specific patch.

--- speech-tools-2.1~release.orig/config/modules/esd_audio.mak
+++ speech-tools-2.1~release/config/modules/esd_audio.mak
@@ -38,13 +38,9 @@
  ##                                                                       ##
  ###########################################################################
 
-INCLUDE_ESD_AUDIO=1
+INCLUDE_ESD_AUDIO=0
 
 MOD_DESC_ESD_AUDIO=Use Enlightenment Speech Demon
 
-AUDIO_DEFINES += -DSUPPORT_ESD
-AUDIO_INCLUDES += -I$(ESD_INCLUDE)
-MODULE_LIBS += -L$(ESD_LIB) -lesd -laudiofile
-MODULE_EXTRA_LIBS += -L$(X11_LIB) -lX11 -lXt
 
 
